src/github/activityBarViewProvider.ts

```diff
@@ -80,6 +80,10 @@ export class PullRequestViewProvider extends WebviewViewBase implements vscode.W
 		}
 	}
 
+	public async refresh(): Promise<void> {
+		this.updatePullRequest(this._item);
```

Comment:
Should this be `await`'d so if a caller wanted to await for the `refresh` method to be called it would work?

Reply:
Added!